phold/README.md

```diff
@@ -1,7 +1,121 @@
 This directory contains code for a PHOLD benchmark implemented using SST. 
+The PHOLD benchmark is based on Fujimoto's 1990 paper [Performance of Time Warp Under Synthetic Workloads](https://gdo149.llnl.gov/attachments/20776356/24674621.pdf).
+PHOLD is a synthetic benchmark developed to surface important simulation characteristics as parameters. 
+This makes it useful for studying the behavior of PDES simulators like SST.
+
+## Prerequisites
+
+It is assumed you have installed [SST version 15.1.0](https://sst-simulator.org/SSTPages/SSTMainDownloads/). 
+You can check which version of `sst` you have installed with the following command: `sst --version`.
+
+## Workflow Overview
+
+The general PHOLD workflow has 4 pieces:
+- The `Node.cpp` and `Node.h` component definitions. These files define the component used in the PHOLD simulation.
+- The `phold_dist.py` simulation description. This is the input that SST expects for the simulation.
+- The `submit.py` and `dispatch.sh` scripts. These allow for the submission of multiple different PHOLD simulations as SLURM jobs, for example running a strong scaling evaluation.
+- The `consolidate.py` script. This consolidates the outputs of the various jobs run by the `submit.py` and `dispatch.sh` scripts into a single `.csv` file for analysis.
+
+Before detailing the various ways these scripts can be used, we review a simple strong scaling workflow.
+
+First, we need to build the PHOLD library so that SST can use them in a simulation:
+```
+make
+```
+This will produce a file, `libphold.so` for use in the simulation.
+
+Next, on a SLURM system, we will submit the jobs for the strong scaling evaluation, using 1, 2, 4, and 8 nodes on a 1000x1000 grid of components.
+Each component will be connected to 2 rings of neighbors (25 connections total), and there will be 2 events moving around the simulation per component.
+We will run the simulation for 100ns. 
```

Comment:
, because that leads to the simulator running for about X minutes on a desktop and eases running sweeps.

Is this the case?  Can you say that or something like it?